src/models/generated/BatteryEMS.jl

```diff
@@ -17,9 +17,8 @@ This file is auto-generated. Do not edit.
         reactive_power::Float64
         reactive_power_limits::Union{Nothing, Min_Max}
         base_power::Float64
+        operation_cost::Union{Nothing, OperationalCost}
```

Comment:
this will be a breaking API change. we need to add a constructor that makes the conversion correctly